F5/single-server-session-iRule.tcl

```diff
@@ -150,6 +150,9 @@ when HTTP_REQUEST {
           or ($path contains "/verse/checksession")
           or ($path contains "/verse/userredirectinfo")
           or ($path contains "/verse/ical"))
+          or ($path contains "/verse/archive"))
+          or ($path contains "/verse/delegation"))
+          or ($path contains "/verse/delegation/archive"))
```

Comment:
@hcl-bpchasse - is this really needed here or is it covered by /verse/delegation above?